app/src/main/res/layout/activity_news_detail.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The UI seems good for starter. If you have time I'd love to have a more nicer UI with more data shown than what's shown in the news list in the 1st screen.